tests/helpers.py

```diff
@@ -6,29 +6,57 @@
 
 from click.testing import CliRunner
 
+from code_annotations.base import BaseSearch, VerboseEcho
 from code_annotations.cli import entry_point
 
 EXIT_CODE_SUCCESS = 0
 EXIT_CODE_FAILURE = -1
 DEFAULT_FAKE_SAFELIST_PATH = 'fake_safelist_path.yaml'
 
 FAKE_CONFIG_FILE = """
-safelist_path: {}
+safelist_path: {fake_safelist_path}
 report_path: test_reports
+source_path: ../
 annotations:
-    pii:
-        - ".. pii::"
+    ".. no_pii::":
+    ".. ignored::":
+        choices: [irrelevant, terrible, silly-silly]
+    "pii_group":
+        - ".. pii::":
         - ".. pii_types::":
             choices: [id, name, other]
         - ".. pii_retirement::":
             choices: [retained, local_api, consumer_api, third_party]
-    nopii: ".. no_pii::"
-    ignored:
-        ".. ignored::":
-            choices: [irrelevant, terrible, silly-silly]
+extensions:
+    python:
+        - py
 """
```

Comment:
While you're in here, why not bake fake_safelist_path into this string right now during module setup?